c-pallets/segment-book/src/lib.rs

```diff
@@ -376,15 +381,16 @@ pub mod pallet {
 								<FailureNumMap<T>>::get(&miner),
 								total_proof,
 								<ConsecutiveFines<T>>::get(&miner),
-							).unwrap_or(());
+							).unwrap_or(()); // Unhandled error
 						} else {
 							<ConsecutiveFines<T>>::remove(&miner);
 						}
 					}
 
-					Self::start_buffer_period_schedule().unwrap_or(());
-					<FailureNumMap<T>>::remove_all(None);
-					<MinerTotalProof<T>>::remove_all(None);
+					Self::start_buffer_period_schedule().unwrap_or(()); // Unhandled error
+					// https://github.com/paritytech/substrate/issues/9002
+					<FailureNumMap<T>>::remove_all(None); // remove_all() deprecated: Use clear() instead
+					<MinerTotalProof<T>>::remove_all(None); // remove_all() deprecated: Use clear() instead
```

Comment:
How should I use `clear()`? I haven't found how to use `clear()`
I saw something like `clear_prefix()` or `remove_all()` Pass in the `Option<u32>`parameter as a limit

Reply:
https://paritytech.github.io/substrate/master/frame_support/storage/types/struct.StorageMap.html#method.clear

Reply:
https://paritytech.github.io/substrate/master/frame_support/storage/types/struct.StorageMap.html#method.clear

Reply:
I used `clear()`, but found no method when compiling. Then I noticed that the version of the framework we are currently using is too backward. It is 2022-04. Sorry, we have no plan to upgrade the framework at present, so I will continue to use the `remove_all()`, but I have calculated the weight for them:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R425-R430
However, this is only temporary. The upgrade framework will be included in our subsequent plans. I have also written a use case for `clear()`:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R645-R669
After we have updated the new framework, I will enable these two methods: `clear_failure_map()` and `clear_total_proof()`